types/reactstrap/v4/lib/Card.d.ts

```diff
@@ -5,6 +5,7 @@ interface Props {
   inverse?: boolean;
   color?: string;
   block?: boolean;
+  body?: boolean;
```

Comment:
The change of block to body is a reactstrap v5 (the base folder) not a v4 change.
The v4 is for bootstrap-alpha and refers to reactstrap v4.